kalshi/src/portfolio.rs

```diff
@@ -1057,49 +1068,70 @@ pub enum OrderType {
     Limit,
 }
 
-
 trait OrderParams {
-    fn get_params(self) -> (
-        Action, 
-        Option<String>, 
-        i32, 
-        Side, 
-        String, 
-        OrderType, 
-        Option<i64>, 
-        Option<i64>, 
-        Option<i64>, 
-        Option<i32>, 
-        Option<i64>
+    fn get_params(
+        self,
+    ) -> (
+        Action,
+        Option<String>,
+        i32,
+        Side,
+        String,
+        OrderType,
+        Option<i64>,
+        Option<i64>,
+        Option<i64>,
+        Option<i32>,
+        Option<i64>,
     );
 }
 
-impl OrderParams for (Action, Option<String>, i32, Side, String, OrderType, Option<i64>, Option<i64>, Option<i64>, Option<i32>, Option<i64>) {
-    fn get_params(self) -> (
-        Action, 
-        Option<String>, 
-        i32, 
-        Side, 
-        String, 
-        OrderType, 
-        Option<i64>, 
-        Option<i64>, 
-        Option<i64>, 
-        Option<i32>, 
-        Option<i64>
+impl OrderParams
+    for (
+        Action,
+        Option<String>,
+        i32,
+        Side,
+        String,
+        OrderType,
+        Option<i64>,
+        Option<i64>,
+        Option<i64>,
+        Option<i32>,
+        Option<i64>,
+    )
+{
+    fn get_params(
+        self,
+    ) -> (
+        Action,
+        Option<String>,
+        i32,
+        Side,
+        String,
+        OrderType,
+        Option<i64>,
+        Option<i64>,
+        Option<i64>,
+        Option<i32>,
+        Option<i64>,
     ) {
         (
-            self.0,
-            self.1,
-            self.2,
-            self.3,
-            self.4,
-            self.5,
-            self.6,
-            self.7,
-            self.8,
-            self.9,
-            self.10
+            self.0, self.1, self.2, self.3, self.4, self.5, self.6, self.7, self.8, self.9, self.10,
         )
     }
 }
+
+#[cfg(test)]
+mod test {
+    use crate::portfolio::MultipleOrderResponse;
+
+    #[test]
+    fn test_serialize_multiple_order_response() -> serde_json::Result<()> {
+        let json = r#"{"orders":[],"cursor":""}"#;
+        let result = serde_json::from_str::<MultipleOrderResponse>(json)?;
+        assert!(result.orders.is_empty());
+        assert!(result.cursor.is_none());
+        Ok(())
+    }
+}
```

Comment:
@dpeachpeach Sorry, the diff is hard to parse because of all the formatting changes. This is the critical part of the PR -- the rest should just be whitespace manipulation.